test/metabase/api/card_test.clj

```diff
@@ -1208,7 +1208,8 @@
                    :collection_id          (u/the-id collection)
                    :collection             (into {:is_personal false} collection)
                    :result_metadata        (mt/obj->json->obj (:result_metadata card))
-                   :metabase_version       config/mb-version-string})
+                   :metabase_version       config/mb-version-string
+                   :dataset_query_metrics_v2_migration_backup nil})
```

Comment:
Instead of busting the indentation here (which someone will have to fix at some point in the future) why don't you just change the `is` stuff below from `=` to `=?`. That way the next person who adds a new column doesn't have to do this too